plugin.cpp

```diff
@@ -190,10 +190,10 @@ static const char *default_config = QUOTE({
             "order" : "19"
             },
     "miBlockSize" : {
-            "description" : "The number of MonitoredItem to be added on to the OPCUA server in single call" ,
+            "description" : "The number of MonitoredItems to be registered with the OPC/UA server in single call to the S2OPCUA OPCUA Toolkit" ,
             "type" : "integer",
             "default" : "100",
-	    "minimum" : "0",
+	    "minimum" : "1",
```

Comment:
I would suggest putting some maximum on this, otherwise very large blocks of memory will be allocated potentially.  Maybe 500 or 1000 would be a reasonable upper limit.